src/storage/storage_manager.cpp

```diff
@@ -127,6 +127,11 @@ void StorageManager::dropTable(table_id_t tableID) {
 
 void StorageManager::prepareCommit(transaction::Transaction* transaction) {
     transaction->getLocalStorage()->prepareCommit();
+    for (auto tableID : wal->getUpdatedTables()) {
```

Comment:
Can you add a brief comment here to explain the changes? Better for readability.